.changeset/trace-intelligence-ui-audit.md

```
@@ -0,0 +1,6 @@
+---
+'@mastra/playground-ui': patch
+'@internal/playground': patch
+---
+
+Reworked the Trace Intelligence views for clarity. The flow chart now has sortable column headers with signal descriptions on hover, horizontal SIGNALS/THEMES section rules, and continuous node-and-ribbon movement when columns are reordered; the bottom distribution rail and stage legend are gone, and clicking a theme opens its details while isolating it in the flow. Compare uses two identical movable points instead of A/B markers and reports deltas as percentages. Lifelines rows fill the area under each theme's share and show instant point tooltips. The theme details panel gains page-numbered examples, a plain-language share sentence, a hue-colored signal heading, and a Trend section with a trace-count-over-time chart replacing the clustering-state history list. Each view explains itself with a one-line description and an info-icon tooltip.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)